compiler/showTypes.bal

```diff
@@ -39,4 +41,37 @@ function subtypeRels(front:SourcePart[] sources) returns string[]|err:Any|io:Err
         order by s
         select s;
     return rels;
-}
\ No newline at end of file
+}
+
+function testSubtypes(front:SourcePart[] sources, string[] expected) returns boolean|err:Syntax|err:Any|io:Error {
+    var [env, m] = check front:typesFromString(sources);
+    var tc = t:typeContext(env);
+    foreach var item in expected {
+        s:SubtypeTest test = check s:parseTypeTest(item);
+        var [t1, t2] = extractSemtypes(test.left, test.right, m);
+        match test.op { 
+            "<" => {
+                if !t:isSubtype(tc, t1, t2) || t:isSubtype(tc, t2, t1) {
+                    return false;
+                }
```

Comment:
This should be an `test:assertFalse` and a `test:assertTrue`.

Reply:
is there a disadvantage/limitation in carrying over a boolean to testSuite and asserting there?

Reply:
When something fails, we need to know which exact check that failed. If we return a boolean for here and assert else were, we'll just get to know something went wrong, but not the exact details.

In fact it's a good idea to write couple of failing tests and see the error messages are meaningful.

Reply:
Did  you test a filing case with multiple assertions? do we get a good error message?